src/components/NcContent/NcContent.vue

```diff
@@ -188,6 +188,11 @@ function setAppNavigation(value: boolean): void {
 	height: 100vh;
 	padding: var(--body-container-margin)!important;
 	backdrop-filter: brightness(50%);
+
+	// Flip the image in RTL mode
+	.vue-skip-actions__image:dir(rtl) {
+		transform: rotateY(180deg);
+	}
```

Comment:
Should be moved below to the `scoped` styles

Reply:
Does it matter? 🤭

Reply:
It's a bit unclear why it is for :focus-within only

Reply:
We can avoid duplication With nesting (in both `<style>` and `<style scoped>`)

Reply:
> We can avoid duplication With nesting

Which is what I did ? :raised_eyebrow: 
I was referring to Ferdinand asking to move my code to the scoped section

Reply:
Yes but that is not needed, you do not need the `:focus-within` for the image mirroring - it can be applied all the time

Reply:
Done